### PR TITLE
ADD: Service Provider Details

### LINKED ISSUE #219: ADD: Service Provider Details
For each package README add Info about Service Providers, with links to API docs, and info about API Rate Limits.

Also add extra lines for extra service providers (eg EthPlorer for Token balances)

----
## Service Providers

This package uses the following service providers:

| Function | Service | Notes|
|---|---|---|
Balances | SoChain | https://sochain.com/api#get-received-value
Transaction History | SoChain |  https://sochain.com/api#get-tx-inputs
Tx Broadcast | SoChain | https://sochain.com/api#send-transaction
Explorer | Blocksteam | https://blockstream.info

Rate limits: https://sochain.com/api#rate-limits

----

- [ ] Bitcoin
- [ ] Ethereum
- [ ] Litecoin
- [ ] BitcoinCash
- [ ] BNB
- [ ] THOR

### PR DESCRIPTION
**Pull request by junkai121**
Fix #219

### CODE REVIEW

File: packages/xchain-cosmos/README.md [hunk 1]
```diff
@@ -20,3 +20,15 @@ yarn add cosmos-client
...
+
+This package uses the following service providers:
+
+| Function                    | Service    | Notes                                                               |
+| --------------------------- | ---------- | ------------------------------------------------------------------- |
+| Balances                    | Cosmos RPC | https://cosmos.network/rpc/v0.37.9 (`GET /bank/balances/{address}`) |
+| Transaction history         | Cosmos RPC | https://cosmos.network/rpc/v0.37.9 (`GET /txs`)                     |
+| Transaction details by hash | Cosmos RPC | https://cosmos.network/rpc/v0.37.9 (`GET /txs/{hash}`)              |
+| Transaction broadcast       | Cosmos RPC | https://cosmos.network/rpc/v0.37.9 (`POST /txs`)                    |
+| Explorer                    | BigDipper  | https://cosmos.bigdipper.live                                       |
```
Comment (veado) [inline packages/xchain-cosmos/README.md:34]: Same here?
```
 Rate limits: No
 ```

File: packages/xchain-bitcoincash/README.md [hunk 2]
```diff
@@ -9,3 +9,15 @@
...
+
+This package uses the following service providers:
+
+| Function                    | Service        | Notes                                                 |
+| --------------------------- | -------------- | ----------------------------------------------------- |
+| Balances                    | Haskoin        | https://api.haskoin.com/#/Address/getBalance          |
+| Transaction history         | Haskoin        | https://api.haskoin.com/#/Address/getAddressTxsFull   |
+| Transaction details by hash | Haskoin        | https://api.haskoin.com/#/Transaction/getTransaction  |
+| Transaction broadcast       | Haskoin        | https://api.haskoin.com/#/Transaction/postTransaction |
+| Explorer                    | Blockchain.com | https://www.blockchain.com                            |
```
Comment (veado) [inline packages/xchain-bitcoincash/README.md:23]: If we don't have rate limits here, I would recommend to add following line (or something like that):
```
 Rate limits: No
 ```

File: packages/xchain-polkadot/README.md [hunk 3]
```diff
@@ -20,3 +20,18 @@ Following peer dependencies have to be installed into your project. These are no
...
 yarn add @polkadot/api
 ```
+
+## Service Providers
+
+This package uses the following service providers:
+
+| Function                    | Service          | Notes                                                                                                       |
+| --------------------------- | ---------------- | ----------------------------------------------------------------------------------------------------------- |
+| Balances                    | Subscan          | https://documenter.getpostman.com/view/1618960/TVCe1oRU?version=latest#38c96f38-0278-4510-a030-a3e6be935ee5 |
```
Comment (veado) [inline packages/xchain-polkadot/README.md:30]: Still Subscan by requesting `https://documenter.getpostman.com/...`?

File: packages/xchain-thorchain/README.md [hunk 4]
```diff
@@ -20,3 +20,15 @@ yarn add cosmos-client
...
+
+This package uses the following service providers:
+
+| Function                    | Service        | Notes                                                               |
+| --------------------------- | -------------- | ------------------------------------------------------------------- |
+| Balances                    | Cosmos RPC     | https://cosmos.network/rpc/v0.37.9 (`GET /bank/balances/{address}`) |
+| Transaction history         | Tendermint RPC | https://docs.tendermint.com/master/rpc/#/Info/tx_search             |
+| Transaction details by hash | Cosmos RPC     | https://cosmos.network/rpc/v0.37.9 (`GET /txs/{hash}`)              |
+| Transaction broadcast       | Cosmos RPC     | https://cosmos.network/rpc/v0.37.9 (`POST /txs`)                    |
+| Explorer                    | Thorchain.net  | https://thorchain.net                                               |
```
Comment (veado) [inline packages/xchain-thorchain/README.md:34]: Same here?
```
 Rate limits: No
 ```

**Review by veado — CHANGES_REQUESTED** [review]

File: packages/xchain-polkadot/README.md (hunk 3 above)
Comment (junkai121) [reply packages/xchain-polkadot/README.md:30]: https://documenter.getpostman.com/view is the API reference for `getBalances`

**Review by veado — APPROVED** [review]

**Pull request merged**